docs/Updating-gene-and-gene_alias-tables.md

```diff
@@ -55,11 +55,9 @@ SELECT count(*) FROM cbioportal.gene;
 SELECT count(*) FROM cbioportal.gene_alias;
 ```
 
-8- Additionally, there are several other tables you may want to update now (only in human).
+8- Additionally, there are other tables you may want to update now (only in human).
 
-1. Updating the pfam graphics if you want to see 3D structures, explained in [this document](Updating-pfam_graphics-table.md)
```

Comment:
this is no longer required, since all this data lives in Genome Nexus now as far as I know